installer/client/cli/utils.py

```diff
@@ -1,6 +1,6 @@
 import requests
 import os
-from openai import OpenAI
+import litellm
```

Comment:
So the switch from the `openai` package for the `cli` alltogether? If so, please remove it from the `cli` group dependencies by running:
```bash
poetry remove --group=cli openai
```

Reply:
Wouldn't we rather leave LiteLLM just right where it is now and remove openai from both groups? Where is the advantage of defining openai separately in the first place? I guess we could have different implemenations in the cli and the server. But are we really going to?

Reply:
I agree. I'll submit a PR to unify all dependencies later then. It should keep things simpler now that we have a single poetry project for everything.

Reply:
Great 😁 Consolidation is always welcome 😅